client/src/components/activities/ActivityFilter/lib/filter-name.ts

```diff
@@ -14,13 +15,11 @@ export const namePredicates: Record<
 	endsWith: (name, value) => name.endsWith(value),
 };
 
-export const nameTypeOptions = Object.keys(
-	namePredicates
-) as ActivityFilterNameType[];
+export const nameTypeOptions = Object.keys(namePredicates) as NameType[];
```

Comment:
Is this still used? I made the select element use `ComboboxItems` (values and labels) so I'm not sure if this is still needed.